tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -227,7 +227,7 @@ def test_points(self):
             self.assertEqual(pnt.geom_typeid, 0)
             self.assertEqual(p.x, pnt.x)
             self.assertEqual(p.y, pnt.y)
-            self.assertEqual(True, pnt == fromstr(p.wkt))
+            self.assertEqual(pnt, fromstr(p.wkt))
             self.assertEqual(False, pnt == prev)
```

Comment:
should we add a comment about why we're not using assertNotEqual here and elsewhere in this file (If there's a reason)?